Userland/Games/Minesweeper/CustomGameDialog.cpp

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
```

Comment:
Obviously it's up to you, but feel free to put your own copyright here (and in the .h file) instead of the generic one. :^)